generators/app/templates/package.json

```diff
@@ -13,6 +13,7 @@
     "start": "node app.js",
     "debug": "node debug app.js",
     "test": "istanbul cover _mocha ./test/**/*.test.js",
-    "console": "sails console"
+    "console": "sails console",
+    "clean": "rm -r -f .tmp"
```

Comment:
Better to use `rm -rf ./.tmp`. So set explicitly that `.tmp` folder under the current directory.